backends/redis.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"gopkg.in/gilmour-libs/gilmour-e-go.v4/protocol"
-	"gopkg.in/redis.v4"
```

Comment:
Again, a fork of 4 should be at 4 only no?  Although it doesn't matter to be honest, but logically it sounds so incorrect :)